releases/traefik-ingress.yaml

```diff
@@ -0,0 +1,110 @@
+      # Basic auth to protect all the routes. Can use htpasswd to generate passwords
+      # > htpasswd -n -b testuser testpass
+      # > testuser:$apr1$JXRA7j2s$LpVns9vsme8FHN0r.aSt11
+      auth: {}
+      # basic:
+      #   testuser: $apr1$JXRA7j2s$LpVns9vsme8FHN0r.aSt11
+
+    kvprovider:
+      ## If you want to run Traefik in HA mode, you will need to setup a KV Provider. Therefore you can choose one of
+      ## * etcd
+      ## * consul
+      ## * boltdb
+      ## * zookeeper
+      ##
+      ## ref: https://docs.traefik.io/user-guide/cluster/
+
+      ## storeAcme has to be enabled to support HA Support using acme, but at least one kvprovider is needed
+      storeAcme: false
+      importAcme: false
+    gzip:
+      enabled: true
+    traefikLogFormat: json
+    rbac:
+      enabled: {{ env "RBAC_ENABLED" | default "false" }}
+    deployment:
+      hostPort:
+        httpEnabled: false
```

Comment:
everything is disabled? (might be an incorrect question, just looks strange :)

Reply:
This is default values.